biocrnpyler/core/component.py

```diff
@@ -85,21 +93,54 @@ def initial_concentration(self, initial_concentration):
     
         self._initial_concentration = initial_concentration
 
+    @property
+    def compartment(self):
```

Comment:
I notice a lot of changes related to compartments. Can you give a short description of the changes in the PR description? Looks like you are adding functionality?

Reply:
You are right -- this is one of the main features that I have added in this PR. Will add more details to the PR description.